content/pt-br/docs/setup/production-environment/tools/kops.md

```diff
@@ -0,0 +1,202 @@
+
+Portanto, normalmente você tem um bucket do S3 para cada equipe de operações (e geralmente o nome corresponderá ao nome da zona hospedada acima!)
+
+Em nosso exemplo, escolhemos `dev.example.com` como nossa zona hospedada, então vamos escolher `clusters.dev.example.com` como o nome do bucket do S3.
+
+* Exportar `AWS_PROFILE` (se precisar selecionar um perfil para que a AWS CLI funcione)
+
+* Crie o bucket do S3 usando `aws s3 mb s3://clusters.dev.example.com`
+
+* Você pode `export KOPS_STATE_STORE=s3://clusters.dev.example.com` e, em seguida, o kops usará esse local por padrão. Sugerimos colocar isso em seu perfil bash ou similar.
+
+### (4/5) Crie sua configuração de cluster
+
+Execute `kops create cluster` para criar sua configuração de cluster:
+
+`kops create cluster --zones=us-east-1c useast1.dev.example.com`
+
+kops criará a configuração para seu cluster. Observe que ele _apenas_ cria a configuração, na verdade não cria os recursos de nuvem - você fará isso na próxima etapa com um arquivo `kops update cluster`.  Isso lhe dá a oportunidade de revisar a configuração ou alterá-la.
+
+Ele exibe comandos que você pode usar para explorar mais:
+
+* Liste seus clusters com: `kops get cluster`
+* Edite este cluster com: `kops edit cluster useast1.dev.example.com`
+* Edite seu grupo de instâncias de nós: `kops edit ig --name=useast1.dev.example.com nodes`
+* Edite seu grupo de instâncias principal: `kops edit ig --name=useast1.dev.example.com master-us-east-1c`
+
+Se esta é sua primeira vez usando kops, gaste alguns minutos para experimentá-los! Um grupo de instâncias é um conjunto de instâncias que serão registradas como nós do kubernetes. Na AWS, isso é implementado por meio de grupos de auto-scaling.
+Você pode ter vários grupos de instâncias, por exemplo, se quiser nós que sejam uma combinação de instâncias spot e sob demanda ou instâncias de GPU e não GPU.
+
+### (5/5) Crie o cluster na AWS
+
+Execute `kops update cluster` para criar seu cluster na AWS:
+
+`kops update cluster useast1.dev.example.com --yes`
+
+Isso leva alguns segundos para ser executado, mas seu cluster provavelmente levará alguns minutos para estar realmente pronto.
+`kops update cluster` será a ferramenta que você usará sempre que alterar a configuração do seu cluster; ele aplica as alterações que você fez na configuração ao seu cluster - reconfigurando AWS ou kubernetes conforme necessário.
+
+Por exemplo, depois de você `kops edit ig nodes`, em seguida `kops update cluster --yes` para aplicar sua configuração e, às vezes, você também precisará `kops rolling-update cluster` para implementar a configuração imediatamente.
```

Comment:
The translation is very nice, but in Portuguese I missed something "depois de você  `kops edit ig nodes`, em seguida `kops update cluster --yes`". Think about "depois de você executar ..., em seguida execute"

Reply:
You're right; thanks for the suggestion!